paper-autocomplete-chips.js

```diff
@@ -365,27 +365,37 @@ class PaperAutocompleteChips extends translatable(PolymerElement) {
 	 */
 	_clearItemSelection(event) {
 		const item = event.model.item,
-			selectedIndex = this.selectedItems.indexOf(item);
-
+			tempIndex = this.selectedItems.findIndex(chip => item[this.textProperty] === chip[this.textProperty]),
```

Comment:
Ah now I understand this a bit better too and my comment was a bit off the last round maybe.
I think we should, in the same way as "enrichChips", check if item is an object or not and then look for it among the selections:
```suggestion
			let selectedIndex = this.selectedItems.indexOf(chip.description);
			if (typeof item === 'object') {
						selectedIndex = this.selectedItems.findIndex(item => item[this.textProperty] === chip.description);
			}
```
I was also a bit tripped up by the naming. "item" in the current code actually refers to the chip, and "chip" actually referes to the item, so this should be changed too.